collectors/apps.plugin/apps_plugin.c

```diff
@@ -107,6 +107,26 @@ static int
 static char *user_config_dir = CONFIG_DIR;
 static char *stock_config_dir = LIBCONFIG_DIR;
 
+// some variables for keeping track of processes count by states
+#define NETDATA_CHART_PRIO_SYSTEM_PROCESS_STATES 601
+typedef enum {
+    RUNNING = 0,
+    SLEEPING, // interruptible sleep
+    SLEEPING_D, // uninterruptible sleep
+    ZOMBIE,
+    STOPPED,
+    END,
+} proc_state;
+
+static int proc_state_count[5];
+static const char *proc_states[] = {
+    [RUNNING] = "running",
+    [SLEEPING] = "sleeping(interruptible)",
```

Comment:
I think the dimension name will be more readable if there is a space before the parenthesis.

Reply:
sure